src/rules/no_inner_declarations.rs

```diff
@@ -6,6 +6,7 @@ use swc_ecmascript::ast;
 use swc_ecmascript::visit::noop_visit_type;
 use swc_ecmascript::visit::Node;
 use swc_ecmascript::visit::Visit;
+// use swc_ecmascript::visit::VisitAll;
```

Comment:
Maybe you forgot to delete this, or intended to replace `Visit` with `VisitAll`?

Reply:
Forgot to delete